salt/modules/junos.py

```diff
@@ -202,7 +204,7 @@ def rpc(cmd=None, dest=None, format='xml', **kwargs):
                 log.warning(
                     'Filter ignored as it is only used with "get-config" rpc')
             xml_reply = getattr(conn.rpc, cmd.replace('-', '_'))(**op)
-            ret['message'] = jxmlease.parse(etree.tostring(xml_reply))
+            ret['rpc-reply'] = jxmlease.parse(etree.tostring(xml_reply))
```

Comment:
Can we make it rpc_reply.
https://github.com/Juniper/ansible-junos-stdlib/pull/192
your code looks to have a bug. We should check the xml_reply type before doing etree.tostring. 
for example if user passes format as json, this would break